Sources/PackageModel/UserToolchain.swift

```diff
@@ -492,16 +540,31 @@ public final class UserToolchain: Toolchain {
                         // architecture subdirectory in `bin` if available.
                         let implib = try AbsolutePath(
                             validating: "usr/lib/swift/windows/XCTest.lib",
-                            relativeTo: installation
+                            relativeTo: XCTestInstallation
                         )
                         if fileSystem.exists(implib) {
                             xctest.append(contentsOf: ["-L", implib.parentDirectory.pathString])
                         }
 
+                        let swiftTestingInstallation: AbsolutePath =
+                            platform.appending("Developer")
+                                .appending("Library")
+                                .appending("Testing-\(info.defaults.swiftTestingVersion)")
+
+                        for flag in ["-I", "-L"] {
+                            swiftTesting += try [
+                                flag,
+                                AbsolutePath(
+                                    validating: "usr/lib/swift/windows/\(triple.archName)",
```

Comment:
`-I` is `usr/lib/swift/windows` not in the "arch" directory

Reply:
Fixed, thank you!